include/alpaka/core/ConcurrentExecPool.hpp

```diff
@@ -589,6 +589,7 @@ namespace alpaka
 
                     auto future(pTaskPackage->m_Promise.get_future());
 
+                    std::lock_guard<TMutex> lock(m_mtxWakeup);
                     m_qTasks.push(std::move(upTaskPackage));
 
                     m_cvWakeup.notify_one();
```

Comment:
Could you please add `{` and `}` around lines 592 to 595 to make it stand out better?